dulwich/porcelain.py

```diff
@@ -303,10 +302,10 @@ def check_diverged(repo, current_sha, new_sha):
 
 
 def archive(
-    repo,
-    committish=None,
-    outstream=default_bytes_out_stream,
-    errstream=default_bytes_err_stream,
+        repo,
```

Comment:
Please don't include formatting changes in the same PR, it makes it harder to review.

Reply:
Sorry, PyCharm auto-formatted.  Do you want me to remove these changes?

Reply:
Done.